pkg/executor/show.go

```diff
@@ -1429,7 +1429,7 @@ func constructResultOfShowCreateTable(ctx sessionctx.Context, dbName *pmodel.CIS
 			restoreCtx.WriteKeyWord("TTL_JOB_INTERVAL")
 			restoreCtx.WritePlain("=")
 			if len(tableInfo.TTLInfo.JobInterval) == 0 {
-				restoreCtx.WriteString(model.DefaultJobInterval.String())
+				restoreCtx.WriteString(model.DefaultJobIntervalStr)
```

Comment:
We should not use `model.DefaultJobInterval.String()` (return `1h0m0s`) as the string interval value because the unit `s` is not supported in `duration.PartionDuration` @YangKeao